website/src/server.mjs

```diff
@@ -50,6 +51,16 @@ export function makeServer({
             path.join(router.wwwRootPath, classifiedDirectory.path),
             { currentURI: request.path }
           );
+          const $ = cheerio.load(out);
+          $("script")
+            .filter(function () {
+              return (
+                this.children[0].data === "\n  //\n" ||
+                this.children[0].data === "\n      //\n    "
```

Comment:
This check doesn't look robust. What if there are more spaces?

Can we check for `//` regardless of the number of spaces?

Reply:
would you consider the following be robust:
` $("script:contains('//')").remove();`
this will remove all the script tags that contains "//"

Reply:
No. I think that would delete script elements like the following:
```html
<script>
// TODO(strager): Fix this for IE.
fetch("/").then(r => console.log(r));
</script>
```

Reply:
I think the best thing we can do is assign id's to the empty script tags and  use the following:
` $('script[id="emptyScript"]').remove();`
an exact search  won't work because each script has different number of spaces.

Reply:
Instead of checking for the presence of `//`, we should check if there is *only* `//` (and optionally surrounding whitespace).

Reply:
I think this will do exactly what we want:
`this.children[0].data.trim() === "//"`
this will get rid of new lines and compare if the tag has only "//".
I have tested it against many script tags and it seems to work perfectly.